finality-aleph/src/party.rs

```diff
@@ -319,15 +332,25 @@ async fn run_session_as_authority<B, C, BE, SC>(
     spawn_handle
         .0
         .spawn("aleph/consensus_session_aggregator", aggregator_task);
+    spawn_handle
+        .0
+        .spawn("aleph/consensus_session_refresher", refresher_task);
 
     let _ = exit_rx.await;
     info!(target: "afa", "Shutting down authority session {}", session_id.0);
     let _ = exit_member_tx.send(());
```

Comment:
Can we please also log error when these futures return `Err`?

Reply:
Sure, just missed that.

Reply:
Fixed.